blueprints/ember-redux/index.js

```diff
@@ -0,0 +1,18 @@
+/*jshint node:true*/
+module.exports = {
+  description: 'Installation blueprint for ember-redux',
+  normalizeEntityName: function() {},
+
+  afterInstall: function() {
+    return this.addAddonsToProject({
+      packages: [
+        {name: 'ember-browserify', target: '~1.1.11'},
+      ]
+    }).then(function() {
+      return this.addPackagesToProject([
+        {name: 'redux', target: '~3.5.2'},
```

Comment:
ah, well I think this is still a HUGE improvement to make the addon more beginner friendly. Lets merge it anyway :) and because it's not breaking I'll just bump to v1.5.0

sound good?